README.md

```diff
@@ -1,2 +1,23 @@
-# arrowdash
+# Arrow Maintainer Dashboard
+
 Quarto dashboard for simplifying project maintenance and monitoring important metrics for tracking project sustainability.
+
+## Virtual environment
+
+For local development both Python and R need to be installed.
+
+In an active R session, run the following commands to install the dependencies:
+
+```r
+install.packages(c("DT", "dplyr", "ggplot2", "knitr", "purrr", "reticulate", "rmarkdown", "stringr", "tibble", "tidyr"))
```

Comment:
@thisisnic I am sure this can be done with using DESCRIPTION file but the only way I know how to do it is with the use of `devtools`. But I am reluctant to add extra packages if not needed. Do you have any ideas?

Reply:
How about we do it with the `remotes` package which contains the relevant subset of `devtools` to do this, but with fewer dependencies?  Then we say here to call `remotes::install_deps()` in the project root directory and then it'll install all of the dependencies from the `DESCRIPTION` file.

Reply:
Done and tested 👍 

Not sure why the CI is failing though - I rebased and the change only effects the README file. @thisisnic any idea what is wrong? The error is:

```
8/53 [unnamed-chunk-4] 
Error in `py_call_impl()`:
! TypeError: string indices must be integers, not 'str'
Run `reticulate::py_last_error()` for details.
```

Reply:
No idea sorry  - I'd be tempted to ignore the CI failure for now and we can reinvestigate later